docs/beast_development.rst

```diff
@@ -42,35 +42,51 @@ project is:
 Development Install
 ===================
 
-It is much easier to perform development if any changes you make to the code are
-immediately reflected in how the ``beast`` runs (in contrast to needed to perform a
-new install each team). This can be achieved by using a development install.
+It is much easier to develop the code if any changes you make are immediately
+reflected in how the ``beast`` runs (in contrast to performing a new install each
+team). This can be achieved by using a development install. First, create a fork
+of the official ``beast`` repository, and clone it to your local drive:
 
-This is most easily achieved via a pip development install. Navigate to the
-directory in your your ``beast`` repository that contains `setup.py`, and run:
+.. code-block:: console
 
-  .. code-block:: console
+   $ git clone https://github.com/YourName/beast.git
 
-     $ mv beast beast-official
-     $ git clone https://github.com/YourName/beast.git
-     $ mv beast beast-YourName
-     $ mv beast-official beast
+Optionally, you can rename this cloned copy:
 
-- Set the value of the fork's 'upstream' to the official distribution so you
-  can incorporate changes made by others to your development fork. In the clone
-  of your fork, run the following:
+.. code-block:: console
 
-  .. code-block:: console
+   $ git clone https://github.com/YourName/beast.git beast-YourName
+
+Set the value of the fork's 'upstream' to the official distribution so you
+can incorporate changes made by others to your development fork. In the clone
+of your fork, run the following:
+
+.. code-block:: console
+
+   $ git remote add upstream https://github.com/BEAST-Fitting/beast.git
+
+In order to run a development installation, navigate to the directory in your
+``beast`` repository that contains `setup.py`, and run:
+
+.. code-block:: console
+
+   $ pip install -e .
+
+Alternatively, you can perform a development install directly though Python
+with:
+
+.. code-block:: console
+
+   $ python setup.py develop
 
-     $ git remote add upstream https://github.com/BEAST-Fitting/beast.git
 
 Adding Branches
 ===============
 
 - Make sure you are in the directory for your fork of the beast. You will be on
-  branch 'master' by default.
+  branch `master` by default.
 
-- Create and switch to a branch (here named 'beast-dev1'; generally it's good
+- Create and switch to a branch (here named `beast-dev1`; generally it's good

```

Comment:
This sentence does not seem to relate to the code-block below.  Is there something missing?

Reply:
I think the view here jumps from lines 92 to 113, it actually is followed by the code block
```
  .. code-block:: console

     $ git checkout -b beast-dev1
```